ci_dependencies.sh

```diff
@@ -25,27 +25,16 @@ if ! (command -v dpkg-query >/dev/null 2>&1 && dpkg-query --list libsodium-dev >
        ([ -e "libsodium" ]) \
 ; then
 
-    FOLDER_NAME="libsodium"
+    FOLDER_NAME="libsodium-stable"
 
 if [ -d "$FOLDER_NAME" ]; then
     echo "$FOLDER_NAME already exist. Skipped." >&2
 else
     echo ""
     BASE_PWD=${PWD}
     echo "`date`: INFO: Building prerequisite 'libsodium' from Git repository..." >&2
-    if [ "x$REQUESTED_BRANCH" = "x" ]; then
-        echo "git clone https://github.com/42ity/libsodium.git $FOLDER_NAME"
-        $CI_TIME git clone --quiet --depth 1 https://github.com/42ity/libsodium.git $FOLDER_NAME
-    else
-        if git ls-remote --heads https://github.com/42ity/libsodium.git | grep -q "$REQUESTED_BRANCH"; then
-            echo "git clone -b "$REQUESTED_BRANCH" https://github.com/42ity/libsodium.git $FOLDER_NAME"
-            $CI_TIME git clone --quiet --depth 1 -b "$REQUESTED_BRANCH" https://github.com/42ity/libsodium.git $FOLDER_NAME
-        else
-            echo "$REQUESTED_BRANCH not found for https://github.com/42ity/libsodium.git"
-            echo "git clone https://github.com/42ity/libsodium.git $FOLDER_NAME"
-            $CI_TIME git clone --quiet --depth 1 https://github.com/42ity/libsodium.git $FOLDER_NAME
-        fi
-    fi
+    echo "git clone -b stable https://github.com/42ity/libsodium.git $FOLDER_NAME"
+    $CI_TIME git clone --quiet --depth 1 -b stable https://github.com/42ity/libsodium.git $FOLDER_NAME
```

Comment:
sanity check: Is this a correct change, from REQUESTED_BRANCH to a fixed one? is zproject version in sync?

Reply:
Should be fixed by applying https://github.com/aquette/fty-security-wallet/pull/1